src/modules/RF69.cpp

```diff
@@ -250,6 +261,11 @@ int16_t RF69::startTransmit(uint8_t* data, size_t len, uint8_t addr) {
   // set packet length
   _mod->SPIwriteRegister(RF69_REG_FIFO, len);
 
+  // optionally write packet length
+  if (_packetLengthConfig == RF69_PACKET_FORMAT_VARIABLE) {
+    SPIwriteRegister(RF69_REG_FIFO, len);
```

Comment:
Call `_mod->SPIwriteRegister()`

Reply:
OK. BTW, why some modules have `_mod` and some don't?